src/crimson/osd/osd_operations/internal_client_request.h

```diff
@@ -0,0 +1,47 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include "crimson/osd/osd_operation.h"
+#include "crimson/osd/pg.h"
+
+namespace crimson::osd {
+
+class InternalClientRequest : public OperationT<InternalClientRequest> {
+public:
+  InternalClientRequest(Ref<PG> pg);
```

Comment:
nit, might want to add the `explicit` specifier before `InternalClientRequest`, like
```c++
explicit InternalClientRequest(Ref<PG> pg);
```

Reply:
nit, for using `Ref` might want to `#include "crimson/common/type_helpers.h"`. as none of the explicitly included header file defines `Ref`. and a header file is supposed to be self-contained.

Reply:
Should be fixed now.